learning-progression-with-la-schema.md

```diff
@@ -186,11 +186,11 @@ We are already working with Git. You are reading a Markdown file in a _private_
 
```

Comment:
I suppose I can't comment on lines that I didn't edit.
On line 156, the last sentence isn't clear to me. Which commit of which process?

Reply:
The process of putting potentially modifying the structure of `namespace screensaver` and writing comments on the functions and variables. That will be a series of commits, and there is a last one. That will be the place where they will tag their repositories and generate a release. This is called _code freeze_, but it's getting too much.